agency-api-spec.md

```diff
@@ -287,15 +357,21 @@ these fields might appear for GSA.
 ```
 
 
-### Success Response
+### Responses
+
+Responses should be in `application/json` format and include an appropriate HTTP
+status code.
+
+
+#### Success Response
 
 **Code:** | 200 OK
 :--- |:---
 **Content:** | `{ "id" : 33, "status_tracking_number": "doj-1234" }`
-**Meaning:** | Confirm that the request was created and return an `id` that can uniquely identify the request in the case management system. The (optional) status tracking number can be used by a requester to track a request.
+**Meaning:** | Confirm that the request was created and return an `id` that can uniquely identify the request in the case management system. The status tracking number (required) will be sent to the requester and used to track a request in your case management system.
```

Comment:
We currently log a warning if we receive a 200 response from an agency that is missing an `id`. Since the status tracking number was optional, however, we don't log anything if it's missing from the response. Is there a need for it to be required? Do we know that agencies will always have both? If so, we'll need to update the code on our end to log cases where it's missing.

Also, it says that it's going to be sent to the requester, but we don't send anything to the requester. Is that a future plan? If so, we may want to indicate that.

Reply:
This is just the spec, so agencies building the API understand they need to treat the tracking number as required. This was the feedback I got from OGE, that it should just be required.

It's okay if we don't implement the check on our end right away. It's also fine that we're not sending the tracking number to the requester in the first iteration of this. We just want to set the spec up so that we can build in the functionality when we're ready without having to go back to agencies to add this.

Reply:
👍 Okay that sounds good.